trunk/public/js/facebook-login.js

```diff
@@ -38,22 +37,28 @@ function fbl_loginCheck() {
                 url: fbl.ajaxurl,
                 success: function (data) {
                     if (data && data.success) {
-                        if( data.redirect && data.redirect.length ) {
+                        if ( fbl.current_page && fbl.current_page.length ) {
```

Comment:
We're close! 

You could pass `fbl.current_page` to the ajax call and then have it validate the redirect there, this would make things simpler and actually leverage `redirect_to`, I'm little unsure if redirect_to would ever be used as it is right now